drivers/espi/Kconfig.xec

```diff
@@ -0,0 +1,19 @@
+# Kconfig.xec - Microchip CEC/MEC ESPI configuration options
```

Comment:
CEC/MEC to XEC

Reply:
The CECs don't have the ESPI block. That's why it is better to say XEC in my opinion like we have been doing it.